layouts/shortcodes/cncf-landscape.html

```diff
@@ -57,11 +57,27 @@
 </script>
 {{- end -}}
 <div id="frameHolder">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js"></script>
```

Comment:
You may want to bring this script local to this repo so you don't have this external dependency.